tests/system/conftest.py

```diff
@@ -34,7 +35,20 @@
 import test_utils.prefixer
 
 import bigframes
-from tests.system.utils import convert_pandas_dtypes
+from tests.system.utils import (
+    convert_pandas_dtypes,
+    delete_cloud_function,
```

Comment:
Same here. Import the "utils" package/module, not the individual functions.

Reply:
Done, thanks.